Homework 8.02 part2

sotickmars opened the pull request:
Доработал до конца задание с flex и float.

Сделал фон с картинкой и наверх наложил opacity другим блоком ( через псевдокласс не мог понять почему не вышло..., точнее выходило - но если только применять position.... но этого делать не хотелось).

sotickmars committed:
PRICING OPTION осталось доверстать шрифт в меню

sotickmars committed:
10.02

sotickmars committed:
Закончил блок STYLISH

sotickmars committed:
end home work 10.02

tpotapchik commented on project/homework/assets/styles/main.css:
```diff
@@ -54,4 +54,160 @@ h1::after {
+	margin-top:1px; 
+}
+
+.inner-menu li:nth-of-type(n+4) {
```
it's worth using more simple selectors

tpotapchik commented on project/homework/assets/styles/main.css:
```diff
@@ -54,4 +54,160 @@ h1::after {
+	font-size: 50px;
+	font-weight: bold;
+	padding-bottom: 0px;
+
```
empty line

tpotapchik commented on project/homework/assets/styles/main.css:
```diff
@@ -54,4 +54,160 @@ h1::after {
+
+.wrap-stylish {	
+	height: 300px;
+	background: url(../images/opo.jpg) no-repeat;
```
`opo.jpg`?

tpotapchik commented on project/homework/assets/styles/main.css:
```diff
@@ -54,4 +54,160 @@ h1::after {
+/***********************FOOTER****************/
+
+.wrap-footer {
+	background: rgb(0, 0, 0);
```
black or #000

tpotapchik commented on project/homework/index.html:
```diff
@@ -37,5 +37,66 @@ <h2 class="inner-title">Easy to Customize</h2>
+			<p class="content">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+		</div>
+		<div class="inner-menu">
+			<ul class="menu basic">
```
https://take.ms/yEZmN

sotickmars replied:
но могу делать через UL?

sotickmars replied:
тогда просто выходит поменьше html структура

tpotapchik commented on project/homework/assets/styles/main.css:
```diff
@@ -54,4 +54,160 @@ h1::after {
+/**************************STILISH AXURE DESIGN********/
+
+.wrap-stylish {	
+	height: 300px;
```
no height value

sotickmars committed:
ИСправление после проверки 11.02

Со списками еще не трогал

sotickmars committed:
Полностью переделал весь код под ДЗ на 8.10

Редактирование HTML структуры.
Исправил неккоректную стилизацию.
Добавление Иконочных шрифтов.

tpotapchik commented on project/homework/assets/css/main.css:
```diff
@@ -0,0 +1,277 @@
+}
+.title-awesome {
+	text-align: center;
+	font-size: 48px;
```
please add line-height

tpotapchik commented on project/homework/assets/css/main.css:
```diff
@@ -0,0 +1,277 @@
+	margin: 0 0 35px 0;
+}
+
+.pricing::after {
```
almost the same as `.title-awesome::after`

tpotapchik commented on project/homework/assets/css/main.css:
```diff
@@ -0,0 +1,277 @@
+	width: 65px;
+	background: #ff8c2c;
+}
+
```
empty line

tpotapchik commented on project/homework/assets/css/main.css:
```diff
@@ -0,0 +1,277 @@
+	padding-top: 66px;
+}
+
+.paragraph-conteiner {
```
what's the word `conteiner`

tpotapchik commented on project/homework/assets/css/main.css:
```diff
@@ -0,0 +1,277 @@
+}
+
+.paragraph-conteiner {
+	display: flex;
```
?

tpotapchik commented on project/homework/assets/css/main.css:
```diff
@@ -0,0 +1,277 @@
+
+.menu {
+	margin-top: 2px;
+}
```
please add line

tpotapchik commented on project/homework/assets/css/main.css:
```diff
@@ -0,0 +1,277 @@
+	font-weight: 200;
+}
+
+.stylish:after {
```
see the comment above

tpotapchik commented on project/homework/assets/css/main.css:
```diff
@@ -0,0 +1,277 @@
+
+.wrap-footer-icons {
+	padding-top: 35px;
+	display: flex;
```
no flex here

tpotapchik commented on project/homework/index.html:
```diff
@@ -2,40 +2,139 @@
+					</div>
+					<div class="menu-section">
+						<ul class="menu menu-paragraph-professional">
+							<li>5 gb of space</li>
```
in html not a good way to format the code. usually it's 4 spaces

sotickmars committed:
Последнее исправление

After
